Source/WTF/wtf/UUID.h

```diff
@@ -54,6 +58,11 @@ WTF_MAKE_FAST_ALLOCATED;
     {
         return UUID { generateWeakRandomUUIDVersion4() };
     }
+    
+#ifdef __OBJC__
+    WTF_EXPORT_PRIVATE RetainPtr<NSUUID> createNSUUID() const;
```

Comment:
Can this be an operator NSUUID * instead (like operator NSString * in String, which returns an autoreleased value)?

Reply:
Sure, will change.